standalone/Dockerfile

```diff
@@ -72,6 +72,8 @@ RUN apt-get update \
     tini supervisor iproute2 procps vim \
     # servers dependencies (see `install.sh`)
     build-essential automake pkg-config libtool libltdl-dev python3-dev python3-setuptools python3-pip \
+    # tor
+    tor \
```

Comment:
Not an expert but from what I can remember Tor recommends to [install from their own package repository](https://support.torproject.org/apt/tor-deb-repo/). Maybe this is something we should be doing here as well?

Reply:
Done. Great advise! Will do some testing and then report back once it can be reviewed again.

Reply:
Testing was successful. [Building the image now](https://github.com/joinmarket-webui/joinmarket-webui-docker/actions/runs/2703821256), will take ~1h and then it's ready to go!